### ISSUE TITLE
latte-dock contains a GPL-incompatible font - source rejected by ubuntu

### ISSUE THREAD

**Issue by rikmills** [psifidotos/Latte-Dock#884]
As latte-dock contains a GPL-incompatible (OPL-licensed) font - (tangerine.ttf), latte dock has been rejected by ubuntu archive admins for 18.04.

> Rejected by Steve Langasek: Source embeds and uses at runtime a GPL-incompatible (OPL-Licensed) font; needs an upstream license exception statement or a patch to not special-case the font

latte-dock (0.7.3)

**Comment by psifidotos** [comment]
Ok ... I think we can can have a v0.7.4 for this. The font is used only at the settings window at the text label at top-left corner... I will try to provide a ,png file produced by that text and drop the font totally...

An alternative could be to provide it as .svg file and find out how Latte managed to alter colors for svg files...

**Comment by rikmills** [comment]
Another alternative is that you provide a statement here that you intend the font to be distributed in binaries (.debs etc) as part of the 'work', even though it is not licensed compatibly with the rest of the 'work'. Apparently that should be enough for now (ubuntu feature freeze is Thursday). Though getting rid of non GPL compatible things is not a bad aim.

**Comment by psifidotos** [comment]
> Another alternative is that you provide a statement here that you intend the font to be distributed in binaries (.debs etc) as part of the 'work', even though it is not licensed compatibly with the rest of the 'work'. 

I dont quite like middle grounds and I dont know if this is acceptable from kde projects.

> Apparently that should be enough for now (ubuntu feature freeze is Thursday).

It is too near in order to produce v0.7.4, I wont rush things concerning this

> I'm also not sure of the status of an image made from a font with that license, if you see what I mean. I would have to check.

please, this is important if you can check it out

**Comment by psifidotos** [comment]
according to wikipedia:
"Open Publication License can optionally restrict the distribution of derivative works or to restrict the commercial distribution of paper copies of the work or derivatives of the work,"

I need to communicate this with kde devs and see what we can do...

**Comment by rikmills** [comment]
> I dont quite like middle grounds and I dont know if this is acceptable from kde projects.
Please check with them.
> It is too near in order to produce v0.7.4, I wont rush things concerning this
Fully understand.  As latte would be a leaf package, we may be able to work around the freeze. I shall ask.
> please, this is important if you can check it out
I am asking. Hoping for an answer this evening perhaps.

**Comment by psifidotos** [comment]
> Please check with them.

I have just sent an e-mail at kde development list

**Comment by rikmills** [comment]
I would ask Riddell on IRC as well. Often only on in worktime, so before 5pm UTC.

**Comment by xnox** [comment]
Providing svg/png rendering does not resolve the licensing issue. The request here is to publish Latte-Dock with a new license, GPL with an exception to use/link OPL licensed font. As the incompatibility is due to the choices of the Latte-Dock copyright holders and the license Latte-Dock copyright owners chose to use. Please provide a linking exception similar to e.g. as shown here for openssl https://opensource.stackexchange.com/questions/2233/gpl-v3-with-openssl-exception . Because otherwise, Latte-Dock might be poisoning the font, and forcing the font to become GPL when it cannot.

**Comment by psifidotos** [comment]
@jriddell my concern based on @xnox  is if that this font can be distributed through Latte code, as such is the case at: https://github.com/psifidotos/Latte-Dock/tree/master/shell/package/contents/fonts

so @xnox I think proposes:

1. keep the font in Latte code and update the Latte License **OR**
2. keep the License as it is and remove the font from Latte source code. In case the font exists in user's system it will be shown correctly otherwise it will fallback.

**Comment by psifidotos** [comment]
the discussion takes place for the following:

![default](https://user-images.githubusercontent.com/2216474/36737207-335104e8-1be3-11e8-80bd-3cda519587e2.png)

the Label using the font is the "atte". This is why  I proposed (3) choice to remove the font totally from Latte code and the mentioned text to be a .png file.

But I dont understand why a png file doesnt solve the issue. The font wouldnt be distributed in such case so its LICENSE shouldnt be violated...

**Comment by jriddell** [comment]
Shipping a an OFL file alongside GPL files does not make it a derived work so that will have no effect.  Probably the easiest way to get round this bureaucracy is for the Ubuntu package not to ship the .ttf file in its .deb.  Shipping a PNG would cause problems with translations and different font preferences & accessibility needs so I suggest not to but the rules at this stage are arbitrary and it would need discussion with the rule makers whether this would get past them or not.

**Comment by perezmeyer** [comment]
With my Debian maintainer hat on:

- Shipping a font just to render a logo is overkill.
- Shipping a png of the font, if the font license allows it and the final result is license-compatible with the remainaing of the code, should be just fine **in this specific case** because it's a logo and thus not translatable.

**Comment by jriddell** [comment]
Using a font to generate an image does not affect the licence of that image.  Else every book ever would be illegal.  (incidently latte-dock runs fine with the font file removed incase anyone considers that a measure of a derived work.)

**Comment by psifidotos** [comment]
> Shipping a png of the font, if the font license allows it and the final result is license-compatible with the remainaing of the code, should be just fine in this specific case because it's a logo and thus not translatable.

based on the OFL which is the Tangerine case:

>The OFL allows the licensed fonts to be used, studied, modified and redistributed freely as long as they are not sold by themselves. The fonts, including any derivative works, can be bundled, embedded, redistributed and/or sold with any software provided that any reserved names are not used by derivative works. The fonts and derivatives, however, cannot be released under any other type of license. **The requirement for fonts to remain under this license does not apply to any document created using the fonts or their derivatives.**

1. for me is clear that a **.png** file is a document created from that font and as such OFL doesnt even apply to it....
2. if we keep the font, I agree that the distribution of the font in the Latte source code should note somehow that there is a font that it is under  SIL-OFL  and not under GPL but this is already the case by: https://github.com/psifidotos/Latte-Dock/blob/master/shell/package/contents/fonts/SIL%20Open%20Font%20License.txt

**Comment by sladen** [comment]
The rejection is on the basis of having an "OPL-licensed" font.  Can't see anything in the rejection referring to an SIL-OFL licensed font.

**Comment by psifidotos** [comment]
@rikmills does the ubuntu part participates in the discussion ?

Tangerine has SIL-OFL license and not an OPL...
as appears at: https://www.fontsquirrel.com/fonts/Tangerine

**Comment by ScarlettGatelyClark** [comment]
I personally put the SIL-OFL licence in the debian/copyright file, as from reading it, seems completely acceptable.

**Comment by psifidotos** [comment]
> I personally put the SIL-OFL licence in the debian/copyright file, as from reading it, seems completely acceptable.

@jriddell  I will follow kde guidelines for this. Personally I dont think that having it at the same directory with the font creates any issues, I prefer it actually because that separates that specific file from the rest project...

Based on **SIL-OFL** I believe that Latte doesnt violate anything, I dont understand what ubuntu actually objected... maybe they confused **SIL-OFL** with **OPL**

**Comment by psifidotos** [comment]
@ScarlettGatelyClark I suppose you propose to move the SIL-OFL from: https://github.com/psifidotos/Latte-Dock/tree/master/shell/package/contents/fonts

to a more visible place? like adding it in LICENSE file?

**Comment by xnox** [comment]
A logo, which one has copyright for, and licenses under GPL would be fine, as then the whole tarball and all the works included are GPL.

Rendering 'atte' in text, whilst having 'L' as an svg, seems very a11y unfriendly, as I'm guessing screen-reader will read that as 'ate', rather than 'Latte'.

However, the .svg of also references Tangerine font. I'm not sure if all the curves inside the .svg have been converted to "path". Does `icons/org.kde.latte.plasmoid.svg` render correctly on systems without the font? And what is the license of that logo .svg file?

Imho, it should be checked if the Logo is all paths, or text. Create a wordmark which is the full "L atte", license the wordmark as GPL, drop the font from the distribution tarball, drop loading and setting custom Font, change the title to load the full wordmark, and add the a11y caption "Latte" to the wordmark.

It seems to me a waste of RAM to load a font just to render one word dynamically, which should be an all-path svg, or a tittle text matching the DE default font.

**Comment by psifidotos** [comment]
@varlesh is the creator of the logo for Latte, @varlesh is it GPL?

>  Does icons/org.kde.latte.plasmoid.svg render correctly on systems without the font? 

**of course it does... and it doesnt matter! The font is SIL-OFL so the documents produced by the font can be whatever license its creator wants to!** 

@xnox the rest are just your developer opinions and could be just bugs to be fixed and nothing related to licenses... 

Creating a single trademark file of course it creates issues because afterwards the textColor used needs adjustment based on the background color and text color used by the plasma theme... Currently the Latte logo updates  the "atte" text color according to the plasma theme without any need to adjust anything...

![default](https://user-images.githubusercontent.com/2216474/36747256-370dadf6-1bfd-11e8-93d4-dd6c363bbaec.png)

the issue concerning the screen reader I suppose it could be solved differently...

**Comment by psifidotos** [comment]
concerning also the issue for screen reader... Latte could use http://doc.qt.io/qt-5/qml-qtquick-accessible.html 

for the "atte" could block accessibility:
`Accessible.ignored: true`

and for the "L" logo:
```
    Accessible.role: Accessible.Button
    Accessible.name: "Latte"
    Accessible.description: "show Latte settings"
```

so please let's stick only to the licenses talk....

**Comment by vorlonofportland** [comment]
Sorry, it would appear there was a typo in my message.  The GPL-incompatible license in question is the SIL Open Font License; I don't know how I managed to abbreviate this to "OPL".

**Comment by rikmills** [comment]
OPL was a typo in the reject message apparently. The SIL Open Font License was what was meant.

**Comment by psifidotos** [comment]
@rikmills @vorlonofportland what is the violation of Latte that I dont see then? SIL-OFL is referenced at: https://github.com/psifidotos/Latte-Dock/tree/master/shell/package/contents/fonts

**Comment by rikmills** [comment]
@psifidotos sidestepping this interesting discussion if you can make a statement of the form:

"We the upstream authors intend for anyone to be able to distribute binaries of latte-dock that use the tangerine.tff font, even though this font is under the SIL Open Font License instead of the GPL"

Then we can circumvent this and get it into ubuntu as the current 0.7.3 tarball stands (with a small license file update in the packaging).

**Comment by psifidotos** [comment]
@rikmills where do you want that statement?

**Comment by rikmills** [comment]
@psifidotos even in the comments here is fine, as long as it can be linked to in the /debian/copyright

**Comment by psifidotos** [comment]
@rikmills of course I never had in mind to change the Tangerine license. Let me check also with @jriddell. 

@jriddell the statement that @rikmills is stating isnt the obvious one based on the licences offered from Latte?

**Comment by varlesh** [comment]
@rikmills it's not original tangerine font:
![image](https://user-images.githubusercontent.com/8083855/36749249-a303386c-1c13-11e8-9c5a-d24e2401300d.png)

**Comment by varlesh** [comment]
I'm modified tangerine font, bit it's NOT ORIGINAL font!
See this topic:
https://github.com/psifidotos/Latte-Dock/issues/1

**Comment by psifidotos** [comment]
@varlesh it's not original tangerine font

you mean, you have altered it a bit for the logo? It doesnt matter, as long as the svg provides paths, the logo is a document so you can relicense it... 

You logo(.svg) can be GPL but the modified font file(.ttf file) must remain SIL-OFL.

what is the license of your logo?

@varlesh one more favour, are you able to produce an svg file like a trademark but the "atte" text to be updated based on the plasma theme? :

![default](https://user-images.githubusercontent.com/2216474/36749651-0cb4bcaa-1c04-11e8-89c8-1ec23898682a.png)

**Comment by varlesh** [comment]
@psifidotos 
Yes. "L" - it's modofy tangerine "L" and is considered as not original.... 

But for "atte" maybe use system font?

>what is the license of your logo?

Free. Use as you like :)

**Comment by psifidotos** [comment]
> Free. Use as you like :)

nice! ok let's keep it GPL then... 

> But for "atte" maybe use system font?

yes, it is the original

> @varlesh one more favour, are you able to produce an svg file like a trademark but the "atte" text to be updated based on the plasma theme text color? 

concerning this, can you make it? I know that in Arc you have managed to achieve this and I dont have the time and technique ...

**Comment by varlesh** [comment]
> concerning this, can you make it? I know that in Arc you have managed to achieve this and I dont have the time and technique ...

Ok, but later.  What size use?

**Comment by psifidotos** [comment]
> Ok, but later. What size use?

for >= 64px.

**Comment by psifidotos** [comment]
> Ok, but later. What size use?

@varlesh sorry >=48px.

**Comment by psifidotos** [comment]
Concerning the Tangerine Font included in Latte source code:

` We the upstream authors intend for anyone to be able to distribute binaries of latte-dock that use the tangerine.tff font, even though this font is under the SIL Open Font License instead of the GPL`

**Comment by varlesh** [comment]
@psifidotos Save as SVG:
```
<svg xmlns="http://www.w3.org/2000/svg" id="svg910" width="138" height="64" version="1.1" viewBox="0 0 138 64">
<defs>
    <style type="text/css" id="current-color-scheme">
      .ColorScheme-Text {
        color:#232627;
      }
      </style>
  </defs>
 <g id="layer1" transform="translate(0,-280.06666)">
  <path style="fill:currentColor" class="ColorScheme-Text" d="m 77.293,280.07058 c -0.4028,-0.0212 -0.90054,0.0515 -1.49414,0.22071 -0.79147,0.11278 -1.63844,0.45175 -2.54297,1.01562 -0.79147,0.4511 -1.52668,1.01476 -2.20508,1.69141 -1.58293,2.70661 -3.16707,5.69631 -4.75,8.96679 -1.58293,3.1577 -3.16511,6.48437 -4.74805,9.98047 l -0.14258,0.34766 c -2.41967,0.3776 -4.68833,0.79655 -6.64453,1.28516 -7.6314,1.9064 -12.39733,3.94226 -16.27343,6.82226 1.65279,-2.9776 3.31261,-5.94848 5.40429,-8.48828 0.22614,-0.5639 0.0556,-0.8457 -0.50976,-0.8457 -2.47653,0.1209 -4.18301,1.16413 -5.90625,2.76953 -1.24374,-0.9021 -2.5985,-1.5782 -4.06836,-2.0293 -1.46987,-0.4511 -3.22262,-0.67773 -5.25781,-0.67773 -3.7312,0 -7.29355,0.90273 -10.68555,2.70703 -3.392,1.8044 -6.38775,4.22843 -8.98828,7.27344 -2.60053,2.93209 -4.69246,6.25886 -6.27539,9.98046 -1.46987,3.6088 -2.20508,7.07538 -2.20508,10.79688 0,4.1727 0.68031,7.27479 2.03711,9.3047 1.46986,1.9172 3.22261,2.875 5.25781,2.875 3.16586,-0.6767 6.44394,-2.70665 9.83594,-6.08986 1.72713,-1.7801 3.40768,-3.72447 5.04687,-5.82617 l 0.0605,0.0488 c 1.51141,-1.8454 2.84469,-3.40571 4.08594,-4.82031 -0.44685,1.7598 -0.79963,3.48759 -1.05469,5.18359 -0.22613,2.1427 -0.28103,4.11748 -0.16797,5.92188 0.22614,1.6916 0.62217,3.0436 1.1875,4.0586 0.6784,1.015 1.52537,1.52344 2.54297,1.52344 1.58293,0 3.50625,-0.79034 5.76758,-2.36914 2.37439,-1.57892 4.94786,-4.10452 7.32226,-7.37501 0.11307,-0.4511 10e-4,-0.79013 -0.33789,-1.01563 -0.3392,-0.2255 -0.62347,-0.2255 -0.84961,0 -2.3744,3.0449 -4.49498,5.56402 -5.96484,6.35352 -1.46987,0.6766 -2.4874,0.62143 -3.05273,-0.16797 -0.6784,-1.015 -0.90583,-2.87739 -0.67969,-5.58399 0.14453,-1.2013 0.35116,-2.47406 0.61914,-3.81836 0.98771,-4.8382 2.27718,-10.61426 5.54297,-13.72656 4.14596,-3.8893 8.76025,-6.22312 18.05468,-8.54492 1.31149,-0.3277 2.79876,-0.68278 4.36329,-1.04688 -2.03695,5.289 -3.77805,10.45186 -5.1875,15.47266 -1.58294,5.6387 -2.375,10.31942 -2.375,14.04102 0,2.1427 0.56464,4.00313 1.69531,5.58204 1.24373,1.4661 3.11019,2.19922 5.59765,2.19922 0.79147,0 1.80901,-0.28181 3.05274,-0.84571 1.24373,-0.6767 2.54359,-1.52211 3.90039,-2.53712 1.3568,-1.0149 2.65861,-2.19943 3.90234,-3.55273 1.3568,-1.4661 2.54295,-2.98946 3.56055,-4.56836 0.11307,-0.5639 -6.2e-4,-0.95809 -0.33984,-1.18359 -0.3392,-0.3383 -0.67643,-0.39347 -1.01563,-0.16797 -3.50506,5.1877 -6.72824,7.78125 -9.66797,7.78125 -2.93973,0 -4.41015,-1.91766 -4.41015,-5.75196 0,-3.3833 0.73521,-7.66972 2.20507,-12.85742 1.35512,-4.7827 3.06201,-9.76554 5.08985,-14.93164 7.59707,-1.5812 16.50896,-3.09089 25.0039,-3.55859 -0.008,0.018 -0.0172,0.0347 -0.0254,0.0527 l -1.1875,2.875 c -2.37439,5.9771 -4.35456,11.78503 -5.9375,17.42383 -1.58293,5.6387 -2.37304,10.31941 -2.37304,14.04101 0,2.1427 0.56464,4.00313 1.69531,5.58204 1.24373,1.46611 3.2098,2.19923 5.69727,2.19923 0.79146,0 1.809,-0.28181 3.05273,-0.84571 1.24373,-0.6767 2.54359,-1.52211 3.90039,-2.53712 1.3568,-1.0149 2.65665,-2.19943 3.90039,-3.55273 1.01298,-1.0946 1.93195,-2.22141 2.75586,-3.37891 0.004,-0.01 0.008,-0.01 0.0117,-0.0156 0.17167,-0.2005 0.28029,-0.33411 0.45898,-0.54101 1.90951,-2.2109 4.2068,-4.73758 5.47266,-5.80078 0.43646,-0.3666 0.91107,-0.74544 1.37891,-1.11914 -0.83744,2.39979 -1.2793,3.68083 -1.2793,6.45703 0,3.7215 0.90383,6.54178 2.71289,8.45899 1.92213,1.9172 4.35323,2.875 7.29297,2.875 3.16586,0 6.16357,-0.90263 8.99023,-2.70703 2.93974,-1.91721 5.76928,-4.17183 7.69141,-6.76563 0.22613,-0.4511 0.22613,-0.84725 0,-1.18555 -0.22613,-0.3383 -0.56532,-0.39357 -1.01758,-0.16797 -1.58293,1.9171 -3.79098,3.13782 -5.82617,4.26562 -1.92213,1.1277 -4.01405,1.69141 -6.27539,1.69141 -2.26133,0 -4.01408,-0.9578 -5.25781,-2.875 -1.24374,-1.9172 -1.86719,-4.28617 -1.86719,-7.10547 v -1.55859 c 0.11306,-1.2406 0.22677,-2.18138 0.33984,-3.42188 3.0528,-0.7895 5.93679,-1.63491 8.65039,-2.53711 2.82666,-0.9022 5.31464,-1.8601 7.4629,-2.875 2.14825,-1.12789 3.84415,-2.36935 5.08789,-3.72265 1.24377,-1.35329 1.86525,-2.41729 1.86525,-3.99609 0,-1.9172 -0.73521,-6.11007 -2.20508,-7.57617 -1.46985,-1.5788 -3.27946,-2.36914 -5.42773,-2.36914 -2.93973,0 -5.76492,1.01502 -8.47852,3.04492 -2.60053,2.0299 -4.91988,4.6255 -6.95508,7.7832 -1.76217,2.7341 -3.20814,7.12484 -4.37695,10.86914 -6.35275,3.4915 -14.54153,15.74427 -18.66797,16.13477 -2.93973,0 -4.41016,-1.91766 -4.41016,-5.75196 0,-3.3833 0.73522,-7.66972 2.20508,-12.85742 1.46987,-5.1877 3.33633,-10.59948 5.59766,-16.23828 l 0.98437,-2.45508 c 5.21945,-0.1623 10.14816,-0.27199 14.03516,-0.34179 3.97401,-0.072 6.39466,-0.10832 7.9707,-0.19532 0.88539,0 0.8154,-0.30899 0.71875,-1.15429 -0.49023,-1.13282 -2.62122,-0.98532 -4.24023,-0.94922 -1.40216,0.031 -2.5941,0.0637 -4.48438,0.0977 -3.64105,0.066 -8.22824,0.17025 -13.07812,0.31445 1.42096,-3.1442 2.87059,-6.03011 4.36914,-9.01953 1.92214,-3.60881 3.84545,-6.87839 5.76758,-9.81055 0.11306,-0.33833 -0.0575,-0.50781 -0.50977,-0.50781 -0.45227,-0.11277 -1.07571,-0.0576 -1.86718,0.16797 -0.79147,0.11278 -1.73805,0.45175 -2.64258,1.01562 -0.79147,0.4511 -1.52668,1.01476 -2.20508,1.69141 -1.58293,2.70661 -3.16511,5.69632 -4.74805,8.9668 -1.25663,2.5068 -2.5129,4.94992 -3.76953,7.66992 -8.15254,0.3119 -16.77513,1.00599 -24.38477,1.87109 1.68178,-3.798 3.41617,-7.48803 5.20899,-11.06445 1.92213,-3.60881 3.84349,-6.87839 5.76562,-9.81055 0.11307,-0.33833 -0.0555,-0.50781 -0.50781,-0.50781 -0.11304,-0.0282 -0.23683,-0.0457 -0.37109,-0.0527 z m -52.01367,22.9961 c 2.12413,0 3.96804,0.45463 5.5332,1.36523 1.67695,0.9106 2.74031,2.10614 3.1875,3.58594 -1.56516,3.6423 -3.41102,7.28543 -5.53516,10.92773 -2.01234,3.5285 -4.07898,6.71495 -6.20312,9.56055 -2.12414,2.8456 -4.19469,5.1788 -6.20703,7 -1.90055,1.7073 -3.46555,2.56055 -4.69532,2.56055 -1.22976,0 -2.29117,-0.79518 -3.18554,-2.38868 -0.78258,-1.7074 -1.17383,-3.81416 -1.17383,-6.31835 0,-3.86991 0.50359,-7.39884 1.50976,-10.58594 1.00618,-3.187 2.34649,-5.91891 4.02344,-8.19531 1.67695,-2.39031 3.57899,-4.2109 5.70313,-5.4629 2.23593,-1.3659 4.58344,-2.04882 7.04297,-2.04882 z m 102.24804,2 c 1.37631,0 2.46668,0.49085 3.26954,1.47265 2.26648,4.60052 0.80133,9.149 -2.75391,11.25586 -2.52326,1.4181 -6.53768,2.50789 -12.04297,3.27149 0.91755,-4.6908 2.40819,-10.54487 4.47266,-12.72657 2.06449,-2.1817 4.41673,-3.27343 7.05468,-3.27343 z"/>
 </g>
</svg>

```

**Comment by varlesh** [comment]
preview:
![image](https://user-images.githubusercontent.com/8083855/36752646-15805fec-1c1d-11e8-81ea-a2807bd4361e.png)

**Comment by varlesh** [comment]
another style:
```
<svg xmlns="http://www.w3.org/2000/svg" id="svg910" width="138" height="64" version="1.1" viewBox="0 0 138 64">
 <defs id="defs3">
  <style id="current-color-scheme" type="text/css">
   .ColorScheme-Text { color:#232627; }
  </style>
 </defs>
 <g id="layer1" transform="translate(0,-280.06666)">
  <path style="fill:currentColor" class="ColorScheme-Text" d="m 77.292969,0.00390625 c -0.4028,-0.0212 -0.900541,0.05149309 -1.494141,0.22070313 -0.79147,0.11277997 -1.638439,0.45175488 -2.542969,1.01562502 -0.79147,0.4511 -1.526678,1.0147562 -2.205078,1.6914062 -1.58293,2.70661 -3.16707,5.6963144 -4.75,8.9667974 -0.222554,0.443958 -0.44544,0.916544 -0.667969,1.367187 -3.078257,0.447272 -6.09341,0.999714 -8.966796,1.693359 -8.237185,1.988485 -14.858715,3.822095 -19.185547,8.810547 -1.24374,-0.9021 -2.5985,-1.578197 -4.06836,-2.029297 -1.46987,-0.4511 -3.222622,-0.677734 -5.257812,-0.677734 -3.7312,0 -7.293547,0.902731 -10.685547,2.707031 -3.392,1.8044 -6.387751,4.228428 -8.9882812,7.273438 -2.6005303,2.93209 -4.6924607,6.258868 -6.2753907,9.980469 C 0.73520822,44.632237 0,48.098813 0,51.820312 0,55.993013 0.68030927,59.09509 2.0371094,61.125 3.5069694,63.0422 5.2597219,64 7.2949219,64 c 3.1658601,-0.6767 6.4439371,-2.706634 9.8359371,-6.089844 1.72713,-1.7801 3.407685,-3.724472 5.046875,-5.826172 l 0.06055,0.04883 c 1.51141,-1.845399 2.844688,-3.405712 4.085938,-4.820312 -0.44685,1.7598 -0.799628,3.487594 -1.054688,5.183594 -0.22613,2.1427 -0.281027,4.117475 -0.167969,5.921875 0.226141,1.6916 0.622175,3.043598 1.1875,4.058593 C 26.967454,63.491558 27.814431,64 28.832031,64 c 1.58293,0 3.506248,-0.790341 5.767578,-2.369141 2.37439,-1.57892 4.947866,-4.10451 7.322266,-7.375 0.11307,-0.4511 9.99e-4,-0.790125 -0.337891,-1.015625 -0.3392,-0.2255 -0.623469,-0.2255 -0.849609,0 -2.3744,3.0449 -4.494984,5.564016 -5.964844,6.353516 -1.46987,0.6766 -2.487404,0.621431 -3.052734,-0.167969 -0.6784,-1.015 -0.905828,-2.877384 -0.679688,-5.583984 0.6088,-6.293248 2.731612,-12.654862 5.09375,-18.257813 1.752159,-5.164284 5.010687,-9.098771 9.75586,-11.621093 3.260337,-1.692481 7.176188,-3.048351 11.482422,-4.087891 1.947867,-0.544817 3.879684,-1.031125 5.804687,-1.486328 -0.539784,1.153031 -1.081192,2.297801 -1.621094,3.490234 l -0.142578,0.347656 -1.792968,4.478516 c -2.03695,5.289 -3.778051,10.451856 -5.1875,15.472656 -1.582941,5.6387 -2.375,10.319416 -2.375,14.041016 0,2.1427 0.564642,4.003121 1.695312,5.582031 C 54.99373,63.266881 56.860196,64 59.347656,64 c 0.79147,0 1.809005,-0.281803 3.052735,-0.845703 1.24373,-0.6767 2.54359,-1.522099 3.90039,-2.537109 1.3568,-1.014902 2.658614,-2.199435 3.902344,-3.552735 1.3568,-1.4661 2.542947,-2.989459 3.560547,-4.568359 0.11307,-0.5639 -6.24e-4,-0.958094 -0.339844,-1.183594 -0.3392,-0.3383 -0.676425,-0.393469 -1.015625,-0.167969 -3.50506,5.1877 -6.728239,7.78125 -9.667969,7.78125 -2.93973,0 -4.410156,-1.917653 -4.410156,-5.751953 0,-3.3833 0.735218,-7.669722 2.205078,-12.857422 1.35512,-4.7827 3.062004,-9.76554 5.089844,-14.93164 l 1.572266,-3.945313 c 0.65025,-1.468474 1.316103,-2.90593 1.982422,-4.341797 8.328158,-1.609546 16.558077,-2.472161 25.001953,-2.86914 -0.866684,1.724478 -1.73282,3.46366 -2.59961,5.339843 l -0.953125,2.257813 c -0.008,0.018 -0.01719,0.03473 -0.02539,0.05273 l -1.1875,2.875 c -2.37439,5.9771 -4.35456,11.785028 -5.9375,17.423828 -1.58293,5.6387 -2.373047,10.319416 -2.373047,14.041016 0,2.1427 0.564642,4.003121 1.695312,5.582031 C 84.044511,63.266903 86.010577,64 88.498047,64 c 0.79146,0 1.809004,-0.281803 3.052734,-0.845703 1.24373,-0.6767 2.543591,-1.522099 3.900391,-2.537109 0.0092,-0.0069 0.01815,-0.01459 0.02734,-0.02148 l 12.675788,-7.78125 c 0.0244,3.646126 0.92472,6.419615 2.70898,8.310547 1.92213,1.9172 4.35323,2.875 7.29297,2.875 3.16586,0 6.16357,-0.902631 8.99023,-2.707031 2.93974,-1.91721 5.76928,-4.171825 7.69141,-6.765625 0.22613,-0.4511 0.22613,-0.847247 0,-1.185547 -0.22613,-0.3383 -0.56532,-0.393569 -1.01758,-0.167969 -1.58293,1.9171 -3.79098,3.137825 -5.82617,4.265625 -1.92213,1.1277 -4.01405,1.691406 -6.27539,1.691406 -2.26133,0 -4.01408,-0.9578 -5.25781,-2.875 -1.09356,-2.082833 -1.64672,-4.668233 -1.79688,-7.4375 l 16.44727,-10.097656 c 2.11598,-1.118319 3.79287,-2.346587 5.02344,-3.685547 C 137.37861,33.68188 138,32.617863 138,31.039062 c 0,-1.917199 -0.73521,-6.110071 -2.20508,-7.576171 -1.46985,-1.5788 -3.27946,-2.369141 -5.42773,-2.369141 -2.93973,0 -5.76492,1.015022 -8.47852,3.044922 -2.60053,2.0299 -4.91988,4.625503 -6.95508,7.783203 -2.65448,4.869314 -3.84117,9.244169 -5.5039,14.287109 -0.43211,1.238256 -0.75364,2.183915 -0.96875,3.185547 l -9.146487,5.451172 c -5.116845,3.184054 -11.4408,3.480195 -11.835937,-1.671875 0,-3.3833 0.735218,-7.669722 2.205078,-12.857422 1.46987,-5.1877 3.336326,-10.599481 5.597656,-16.238281 l 0.984375,-2.455078 0.921875,-2.228516 c 0.837945,-1.854144 1.690697,-3.618489 2.552734,-5.367187 4.946226,-0.120862 9.987446,-0.109971 15.189456,-0.01953 1.76201,0.117424 0.83152,-1.21618 -0.25391,-1.30664 -2.40756,-0.220923 -7.38827,-0.626987 -13.8457,-0.867188 0.24313,-0.486124 0.48088,-0.96889 0.72656,-1.458984 1.92214,-3.60881 3.84545,-6.8783869 5.76758,-9.81054688 0.11306,-0.33832999 -0.0575,-0.5078125 -0.50977,-0.5078125 -0.11306,-0.0281925 -0.23668,-0.04557281 -0.37109,-0.05273437 -0.40324,-0.02148469 -0.90249,0.05152561 -1.49609,0.22070313 -0.79147,0.11277997 -1.73805,0.45175487 -2.64258,1.01562502 -0.79147,0.4511 -1.52668,1.0147562 -2.20508,1.6914062 -1.546411,2.644157 -3.092291,5.5676413 -4.638672,8.7500004 -3.867239,-0.06506 -8.038672,-0.05487 -12.341797,0.0957 -3.873557,0.13554 -7.844087,0.391094 -11.777344,0.791015 0.356078,-0.728824 0.70369,-1.473689 1.064453,-2.193359 1.92213,-3.60881 3.843495,-6.8783869 5.765625,-9.81054688 0.11307,-0.33832999 -0.0555,-0.5078125 -0.507813,-0.5078125 C 77.551023,0.02844265 77.427229,0.01090625 77.292969,0.00390625 Z M 25.279297,23 c 2.12413,0 3.968043,0.454634 5.533203,1.365234 1.67695,0.9106 2.74031,2.106138 3.1875,3.585938 -1.56516,3.6423 -3.411016,7.285434 -5.535156,10.927734 -2.01234,3.5285 -4.078985,6.714947 -6.203125,9.560547 -2.12414,2.8456 -4.194692,5.1788 -6.207031,7 C 14.154138,57.146753 12.589145,58 11.359375,58 10.129615,58 9.0681981,57.204828 8.1738281,55.611328 7.3912481,53.903928 7,51.797159 7,49.292969 c 0,-3.86991 0.5035956,-7.398838 1.5097656,-10.585938 1.00618,-3.187 2.3464874,-5.918912 4.0234374,-8.195312 1.67695,-2.39031 3.578985,-4.210891 5.703125,-5.462891 C 20.472258,23.682928 22.819767,23 25.279297,23 Z m 102.248043,2 c 1.37631,0 2.46667,0.490856 3.26954,1.472656 2.26648,4.60052 0.80133,9.149 -2.75391,11.25586 -0.0133,0.0075 -0.0296,0.01403 -0.043,0.02148 l -13.34766,7.955078 c 0.33367,-6.68608 2.72244,-13.790884 5.82032,-17.43164 C 122.53715,26.091737 124.88939,25 127.52734,25 Z" transform="translate(0,280.06666)"/>
 </g>
</svg>
```
preview:
![image](https://user-images.githubusercontent.com/8083855/36753270-1ac13524-1c1f-11e8-89f0-88a3f26a1668.png)

**Comment by psifidotos** [comment]
@varlesh is it possible to make it italic?

**Comment by psifidotos** [comment]
fixed with 0.7.4 release...

https://psifidotos.blogspot.nl/2018/03/latte-bug-fix-release-v074.html

**Comment by rikmills** [comment]
@psifidotos FYI, 0.7.3 is now in ubuntu https://launchpad.net/ubuntu/+source/latte-dock and will update that to 0.7.4 as soon as is practical.
